lib/git-auto-updater.js

```diff
@@ -236,7 +236,7 @@ function scheduleUpdateCheck(error) {
     var minutes = now.getMinutes();
 
     if (hours > conf.time.hours ||
-        (minutes > conf.time.minutes && hours === conf.time.hours)) {
+        (minutes >= conf.time.minutes && hours === conf.time.hours)) {
```

Comment:
Naming things, cache invalidation and off-by-one... Definitely the hardest problems ;)